pkgs/applications/misc/redshift/default.nix

```diff
@@ -62,10 +68,15 @@ let
 
       # the geoclue agent may inspect these paths and expect them to be
       # valid without having the correct $PATH set
-      postInstall = ''
+      postInstall = if (pname == "gammastep") then ''
+        substituteInPlace $out/share/applications/gammastep.desktop \
+          --replace 'Exec=gammastep' "Exec=$out/bin/gammastep"
+        substituteInPlace $out/share/applications/gammastep-indicator.desktop \
+          --replace 'Exec=gammastep-indicator' "Exec=$out/bin/gammastep-indicator"
+      '' else ''
         substituteInPlace $out/share/applications/redshift.desktop \
           --replace 'Exec=redshift' "Exec=$out/bin/redshift"
-        substituteInPlace $out/share/applications/redshift.desktop \
+        substituteInPlace $out/share/applications/redshift-gtk.desktop \
```

Comment:
This change should come in a dedicated commit such as:

```
redshift: fix substitute command for desktop entry
``

Reply:
Oh well, I could consider doing that but I don't really see the point tbh. It's a trivial "fix" and it was working perfectly fine without it. Generally I agree, but I'm not sure if it's really worth a separate commit in this case.

Reply:
Then please add such a comment to the commit message, below `gammastep: init`. And please describe that you reuse `mkRedshift` for `gammastep`.

Reply:
I did split the commits this time.

> And please describe that you reuse mkRedshift for gammastep.

IMO commit messages should always explain why, not how (that's what the patch is for).

Also: I appreciate the feedback, I'm just a bit concerned that we waste important time due to [bikeshedding](https://en.wiktionary.org/wiki/bikeshedding) here as all of these changes don't affect the result. But at least it's better documented in the history now :)